.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -23,8 +23,15 @@ Please also:
 - [ ] This PR meets the Acceptance Criteria in the JIRA story
 - [ ] The JIRA story has been moved to Dev Review
 - [ ] This PR includes appropriate unit tests
-- [ ] I have run and tested this change locally
-- [ ] I have run the E2E tests on ths change against my local UI and/or API server with `yarn test-local` or [yarn test-local-devup](https://github.com/all-of-us/workbench/blob/main/e2e/README.md#examples)
-- [ ] If this includes a UI change, I have taken screen recordings or screenshots of the new behavior and notified the PO and UX designer
-- [ ] If this includes an API change, I have updated the appropriate Swagger definitions and notified API consumers
+- [ ] I have added explanatory comments where the logic is not obvious
+- [ ] I have run and tested this change locally, and my testing process is described here
+- [ ] If this includes a UI change, I have run the E2E tests on ths change against my local UI and/or API server with `yarn test-local` or [yarn test-local-devup](https://github.com/all-of-us/workbench/blob/master/e2e/README.md#examples)
```

Comment:
test-local-devup actually no longer exists, seems I missed some references to that

Reply:
hmm. @aweng98 could you confirm the proper command for running e2e tests against Local->Local?  We may need to also update e2e readme and/or package.json

Reply:
`yarn test-local` is for: `local ui -> local api` and `local ui -> test api`.  `yarn test-local-devup` cmd was removed in [https://github.com/all-of-us/workbench/pull/5544](https://github.com/all-of-us/workbench/pull/5544)

Reply:
e2e readme update https://github.com/all-of-us/workbench/pull/6494